### ISSUE TITLE
Heating Plant

### ISSUE THREAD

**Issue by KiloJKilo** [RealismusModding/FS17_seasons#653]
Would it be possible to adjust the economy of the heating plant based upon the season? 

Example:
Summer: Pays $0
Fall: Early:$10/Mid:$50/Late:$100
Winter: Early:$250/Mid:$500/Late:$350
Spring: Early:$150/Mid:$100/Late:$50

Those prices, of course, are for demonstrative purposes. Just throwing this out there for consumption.

**Comment by reallogger** [comment]
We might do something about this in a later version. For instance making the income dependent on dailyr average air temperature.

**Comment by KiloJKilo** [comment]
Makes sense. You could inversely adjust the windmills. Higher demand for wind energy in summer than winter.

**Comment by reallogger** [comment]
Income from wind mills will be dependent on wind speed. And solar panels on amount of sun ;)

**Comment by KiloJKilo** [comment]
Is there a method to see what the wind speed is, in game?

**Comment by Dzi4d3k** [comment]
@KiloJKilo 
There is still "will be dependent" status, so this is in development :)
